src/renderer/uia/lib/sources

```diff
@@ -0,0 +1,8 @@
+!include ..\sources.inc

```

Comment:
Do we actually need a `sources` for this? will it _ever_ build in the OS repo?

Reply:
Removed.